run-test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+npx hardhat compile
+find test -name "*.test.js" | sort | parallel -k --jobs ${JOBS_NUMBER} --results pout 'npx hardhat test {}'
```

Comment:
Where is this variable defined?

Reply:
It's supposed to be put from outside. But I think it'll be better to get it from the system.

Reply:
Thank you

Reply:
solved

Reply:
You can use default values to allow change number of jobs from outside:
```
DEF_JOBS_NUMBER=$(nproc)
.... parallel -k --jobs ${JOBS_NUMBER:=$DEF_JOBS_NUMBER} ... 
```

Reply:
solved